var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -17,12 +17,19 @@ class PyFenicsDolfinx(PythonPackage):
     version("master", branch="master")
 
     depends_on("cmake@3.9:", type="build")
+    depends_on("pkgconfig", type=("build", "run"))
```

Comment:
This is probably only needed at build-time

Reply:
DOLFINX uses `pkgconfig` to support JIT linking at runtime.